ci-operator/step-registry/openshift/e2e/aws/single-node/openshift-e2e-aws-single-node-workflow.yaml

```diff
@@ -12,5 +12,7 @@ workflow:
     - chain: gather-network
     - chain: gather-core-dump
     - chain: ipi-deprovision
+    env:
+      OPENSHIFT_INSTALL_PRESERVE_BOOTSTRAP: true
```

Comment:
When `OPENSHIFT_INSTALL_PRESERVE_BOOTSTRAP` is used, do the bootstrap resources get cleaned up during an `openshift-installer destroy cluster` operation?

If not, this might leak bootstrap machines

Reply:
It seems to only be respected on ``openshift-installer create cluster``
Also I do see some bootstrap-related resources being deleted on the logs:
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/20592/rehearse-20592-pull-ci-openshift-origin-master-e2e-aws-single-node/1419952737998606336/artifacts/e2e-aws-single-node/ipi-deprovision-deprovision/artifacts/.openshift_install.log

Reply:
I think that destroy is using the same directory with the same TF file so it should destroy the bootstrap if it's still there.